spec/intl.html

```diff
@@ -1736,6 +1753,7 @@ <h1>Temporal.Calendar.prototype.yearMonthFromFields ( _fields_ [ , _options_ ] )
             1. If _calendar_.[[Identifier]] is *"iso8601"*, then
               1. Let _result_ be ? ISOYearMonthFromFields(_fields_, _overflow_).
             1. Else,
+              1. Perform ? CalendarResolveFields(_calendar_.[[Identifier]], _fields_, ~year-month~).
```

Comment:
I was hoping that we didn't need to have different validation steps between dates and year-months, as we do for month-days, and I hadn't yet found any counterexample that would require it. Do you know of one, or is this split just in general?

Reply:
No, in practical implementations I expect CalendarResolveFields to be tightly integrated with CalendarDateToISO/CalendarMonthDayToISOReferenceDate and use identical logic for deciding whether to accept (and translate to ISO 8601) or reject a full date fields bag vs. a year–month fields bag. But this spec organization makes more sense for demonstrating parallels between the required ISO 8601 calendar and its dedicated operations vs. optional calendars with analogous but less-specified operations.